node/src/accountant/db_access_objects/pending_payable_dao.rs

```diff
@@ -65,9 +66,7 @@ impl PendingPayableDao for PendingPayableDaoReal<'_> {
             .filter(|hash| !all_found_records.keys().contains(hash));
         all_found_records
             .iter()
-            .sorted()
             .map(|(hash, rowid)| (Some(*rowid), *hash))
-            .rev()
```

Comment:
Believe me, this was well thought out. We reviewed it with Dan and in fact this _IS_ already code of the master branch, here you're seeing changes from the master merge before the Steve's GH-600. 

Long story short, I had needed the sorting just for a test where I assert on a collection but the ordering was different in times (the set of the elements though stay the same anytime). 

I needed to do something similar for the hotfix I was working on. So I chose to delete this because it was too tight to this low level functionality, instead I did anew but outside and in combination with measures that eventually fixed the emergency issue.